utils/utils.go

```diff
@@ -447,6 +447,38 @@ func CurrencyBalance(
 	return liveAmount, liveBlock, nil
 }
 
+// CurrencyBalanceBatch returns the balance batch of an account
+// for all currencies at a particular height.
+// It is up to the caller to determine if the retrieved
+// block has the expected hash for the requested index.
+func CurrencyBalanceBatch(
```

Comment:
Why this change is related to dest tag?

Reply:
This is related the support of getting all balances of the pre-funded account when currency is nil, the reason we add this is because we use STX to test destination tag and STX rosetta implementation `/account/balance` only support nil currency, which is compliant our rosetta spec. However, out rosetta cli/sdk has the limitation of doesn't support nil currency, add this for both testing STX and scaling rosetta cli/sdk

Reply:
Maybe rename this method to `AllCurrencyBalance`?

Reply:
Sure, will update